### PR TITLE
Add Sketchnote of "Platforms State of the Union" from WWDC21

### PR DESCRIPTION
**Pull request by fbernutz**

### CODE REVIEW

File: content/notes/wwdc21/102.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 ---
-contributors: abadikaka, zntfdr
+contributors: abadikaka, zntfdr, felibe444
 ---
 
+![Sketchnote of WWDC 2021 Platforms State of the Union with detailed announcements about Xcode 13, Xcode Cloud, Swift, SwiftUI, Swift Playground 4, AR, Metal, Focus, Screentime API, Widgets, SharePlay and more][sketchnote]
+
+> This note was originally published at [https://fbernutz.github.io/sketchnotes/wwdc21-state-of-the-union/](https://fbernutz.github.io/sketchnotes/wwdc21-state-of-the-union/)
```
Comment (fbernutz) [inline content/notes/wwdc21/102.md:7]: Is this fine here or should I rephrase it and move it to the bottom of the file?

Comment (zntfdr) [reply content/notes/wwdc21/102.md:7]: For notes where the sketch is an addition to the written notes, please use `Part of this note was originally published at http...`.

I'm planning to add a new feature where this is done automatically (thanks to @mackuba for the suggestion 🙌🏻), however for the moment please update the text. Thank you and I'm sorry for the inconvenience 🙏🏻

Comment (zntfdr) [reply content/notes/wwdc21/102.md:7]: ...and yes please move it to the bottom of the note, thank you!

Comment (fbernutz) [reply content/notes/wwdc21/102.md]: sounds good! done ✅

**Comment by zntfdr** [comment]
Thank you @fbernutz 😃 
[All your notes](https://www.wwdcnotes.com/community/felibe444/) are now live! 👏🏻

**Comment by fbernutz** [comment]
> Thank you @fbernutz 😃
> [All your notes](https://www.wwdcnotes.com/community/felibe444/) are now live! 👏🏻

Yaaay, thank you 🥳

**Pull request merged**